### PR TITLE
DOC: documentation on how to retrieve selected result.

### PR DESCRIPTION
**Pull request by JaydenElliott**
**Problem**
As someone who had never used react-select before, I found it quite difficult to figure out how to obtain the selected results. 

To obtain the results of your select, you can use "selectedProps", however this is not too clear in the current documentation as there is no mention of onSelect/result keywords that would be useful for someone new to react-select.

**Implemented Solution**
I added a few lines of code, copied from the "props" heading, subsection "selectedProps" to the main page to make obtaining results accessible.

Thanks.

### CODE REVIEW

**Comment by Tintef** [comment]
@JaydenElliott Is there any difference with [this section](https://tintef.github.io/react-google-places-autocomplete/docs/examples#controlled-input) of the documentation?

**Comment by JaydenElliott** [comment]
@Tintef No difference, just moved it to the main page as it seems to be a "basic" fundamental usage of your library.

**Comment by nikodunk** [comment]
I second this – took me 20min to find the relevant section of "controlled" aka getting the actual results. Great library otherwise!

**Comment by Ethal-Askander** [comment]
I also thought that this was not very clear.
I think this would be a good merge.

File: docs/docs/basic-usage.mdx [hunk 1]
```diff
@@ -32,8 +34,8 @@ First, generate an `apiKey` in order to use it to load [Google Maps JavaScript A
 ### Use the component
 
 ```js
-import React from 'react';
-import GooglePlacesAutocomplete from 'react-google-places-autocomplete';
+import React from "react";
+import GooglePlacesAutocomplete from "react-google-places-autocomplete";
```
Comment (Tintef) [inline docs/docs/basic-usage.mdx:38]: Please use single quotes, so it's consistent with the rest of the docs :)

File: docs/docs/basic-usage.mdx [hunk 2]
```diff
@@ -44,4 +46,18 @@ const Component = () => (
...
+import ...
+
+const Component = () => (
+  const [value, setValue] = useState(null);
+  <div>
+    <GooglePlacesAutocomplete selectProps={{
+          value,
+          onChange: setValue,
+        }} />
+  </div>
```
Comment (Tintef) [inline docs/docs/basic-usage.mdx:61]: Can you add the return statement?
```suggestion
const Component = () => (
  const [value, setValue] = useState(null);
  
  return (
    <div>
      <GooglePlacesAutocomplete
        selectProps={{
          value,
          onChange: setValue,
        }}
      />
    </div>
  );
}
```

**Review by Tintef — COMMENTED** [review]
@JaydenElliott I was a bit hesitant if this was necessary but given the support from others let's make this be a thing!

**Comment by Tintef** [comment]
Closing due to lack of updates.

Added the change on [this commit](https://github.com/Tintef/react-google-places-autocomplete/commit/67113b89d5aac4bb480b6718017d49859b37fedd)

**Pull request closed without merging**